.github/workflows/auto-merge-deps.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow will auto merge a PR authored by dependabot[bot]. It runs only on open PRs ready for
+# review.
+#
+# It will merge the PR only if: it is authored by dependabot[bot], is a patch semantic update, and
+# all CI checks are successful (ignoring the soon-to-be-removed Jenkins check).
+#
+# The workflow is divided into multiple sequential jobs to allow giving only minimal permissions to
+# the GitHub token passed around.
+name: Dependabot auto-merge updates
+on:
+  pull_request:
+    types:
+      - ready_for_review
+      - opened
+      - reopened
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  metadata:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+    outputs:
+      update-type: ${{ steps.metadata.outputs.update-type }}
+
+  await-checks:
+    runs-on: ubuntu-latest
+    needs:
+      - metadata
```

Comment:
My understanding is that jobs which require another job will be skipped if the first job was skipped as well, so we don't need to keep specifying the previous conditions.